### ISSUE TITLE
Action.<key> Not Routing to Siblings?

### ISSUE THREAD

**Issue by letsgo-steve** [aksonov/react-native-router-flux#1494]
### Version
Tell us which versions you are using: 
````
  "dependencies": {
    "native-base": "^0.5.18",
    "navbar-native": "^1.6.1",
    "react": "15.4.1",
    "react-native": "0.39.2",
    "react-native-drawer": "^2.3.0",
    "react-native-router-flux": "^3.37.0",
    "react-native-vector-icons": "^3.0.0"
  },
````

### Expected behaviour

When navigating to the next scene in a collection of scenes, i.e.

- outerScene
   - innerSceneA
   - innerSceneB
- outerSceneB

I should be able to call Actions.innerSceneB() from innerSceneA. 

### Actual behaviour

When I call Actions.innerSceneB() it seems that the router pushes another copy of A. This is evidenced by the fact that I have to pop N times (where I click my button N times) to get back to the original scene that triggered the sequence.

### Steps to reproduce

1. Create a simple application with nested scenes as per the above.
2. Attempt to navigate to the adjacent scene.
3. The pushed route will instead be another copy of the first scene.

Example routing setup:

````
           <Router>
                <Scene key="root">
                    <Scene key="ImageCover" component={ImageCover}>
                        <Scene key="ImageCoverNavigation" component={NavigationBar}>
                            <Scene key="welcomeScreen" component={WelcomeScreen} title="Welcome Steve" />
                        </Scene>
                    </Scene>
                </Scene>
                <Scene key="journey" component={SequenceNavigation}>
                    <Scene key="search" component={SearchScreen} title="Search" />
                    <Scene key="results" component={ResultsScreen} title="Results"  />
                </Scene>
            </Router>

````

Basically I can't Actions.results() from search. I've also tried stripping off the wrapper component on the journey element, but no dice either.

**Comment by letsgo-steve** [comment]
Problem existed at the keyboard/chair interface layer. Has been resolved.
